tests/compaction_length_refinement.prm

```diff
@@ -110,10 +110,6 @@ subsection Model settings
   set Include melt transport                  = true
 end
 
-subsection Melt settings
-  set Melt transport threshold                = 0
```

Comment:
Could this be what is causing the changes? Or was the melt transport threshold not used before as well?